packages/utils/regl-renderer/types/rendering/commands/drawMesh/vColorShaders.d.ts

```diff
@@ -0,0 +1,3 @@
+declare const vColorFrag: "\nprecision mediump float;\nvarying vec3 surfaceNormal, surfacePosition;\n\nuniform float ambientLightAmount;\nuniform float diffuseLightAmount;\nuniform float specularLightAmount;\n\nuniform vec3 lightDirection;\nuniform vec4 lightColor;\nuniform vec3 opacity;\nuniform float uMaterialShininess;\n\nvarying vec4 vColor;\nuniform vec4 ucolor;\nuniform float vColorToggler;\n\nuniform vec2 printableArea;\nvec4 errorColor = vec4(0.15, 0.15, 0.15, 0.3);//vec4(0.15, 0.15, 0.15, 0.3);\nvarying vec4 _worldSpacePosition;\nvarying float ambientAo;\n\nvoid main () {\n  vec4 depth = gl_FragCoord;\n  vec4 endColor = vColor * vColorToggler + ucolor * (1.0 - vColorToggler);\n\n  vec3 ambient = ambientLightAmount * endColor.rgb ; //ambientAo * \n\n  float diffuseWeight = dot(surfaceNormal, lightDirection);\n  vec3 diffuse = diffuseLightAmount * endColor.rgb * clamp(diffuseWeight , 0.0, 1.0 );\n\n  //specular\n  \n  vec4 specularColor = vec4(lightColor);\n  vec3 eyeDirection = normalize(surfacePosition.xyz);\n  vec3 reflectionDirection = reflect(-lightDirection, surfaceNormal);\n  float specularLightWeight = pow(max(dot(reflectionDirection, eyeDirection), 0.0), uMaterialShininess);\n  vec3 specular = specularColor.rgb * specularLightWeight * specularLightAmount;\n\n  /*float light2Multiplier = 0.2;\n  float diffuseWeight2 = dot(surfaceNormal, vec3(-lightDirection.x, lightDirection.y, lightDirection.z));\n  vec3 diffuse2 = diffuseLightAmount * endColor.rgb * clamp(diffuseWeight2 , 0.0, 1.0 ) * light2Multiplier;\n\n  float light3Multiplier = 0.2;  \n  float diffuseWeight3 = dot(surfaceNormal, vec3(lightDirection.x, -lightDirection.y, lightDirection.z));\n  vec3 diffuse3 = diffuseLightAmount * endColor.rgb * clamp(diffuseWeight3 , 0.0, 1.0 ) * light3Multiplier;\n\n  float light4Multiplier = 0.2;  \n  float diffuseWeight4 = dot(surfaceNormal, vec3(-lightDirection.x, -lightDirection.y, lightDirection.z));\n  vec3 diffuse4 = diffuseLightAmount * endColor.rgb * clamp(diffuseWeight4 , 0.0, 1.0 ) * light4Multiplier;*/\n  \n  gl_FragColor = vec4((ambient + diffuse + specular), endColor.a);\n  //gl_FragColor = vec4((ambient + diffuse + diffuse2 + diffuse3 + diffuse4), endColor.a);\n}\n";
```

Comment:
The redeclaration of these massive string seems wasteful. Are these declarations useful?